extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -85,11 +85,35 @@ public function extendImageData( $data, $width ) {
 			'width' => $dimensions[ 'width' ],
 			'thumbnail' => $thumbnail->getUrl(),
 			'key' => urlencode( $data[ 'key' ] ),
-			'media-type' => $data[ 'isVideo' ] ? 'video' : 'image'
+			'media-type' => $data[ 'isVideo' ] ? 'video' : 'image',
+			'mercuryComponentAttrs' => json_encode( [
```

Comment:
What do you think about moving response encoding to a separate part of code? As is, this public method will return data that contains both raw and encoded elements.